utils/ftw.js

```diff
@@ -151,5 +151,6 @@ module.exports = {
     getProblem,
     getAnswer,
     getStats,
-    resetStats
+    resetStats,
+    sendMessage
```

Comment:
Would rather you use a new `sendMessage` function instead of having to import `ftw.js`.